police_request/PoliceRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+    
+    /**
+     * Example code (commented) at the bottom of this file.
+     * If you have any problems, open an issue on GitHub or
+     * email me at cameron.white@ufl.edu
+     **/
+
+    class PoliceRequest {
+
+        const SEARCH_TYPES = array("All", "Report Number", "Report Date", "Crime Log Data", "Type of Crime", "Location", "Disposition");
+
+        /**
+         * Parses the input HTML and returns an associative array containing crime data.
+         * Indices of the returned arrays are as follow: agency, reportNumber, reportDate, reportTime,
+         * crimeLogDate, crimeType, crimeDate, location, disposition
+         **/
+        function getRows($html){
+            $dom = new DOMDocument();
+            $dom->loadHtml($html);
+            $tables = $dom->getElementsByTagName("table");
+            $rows = array();
+        
+            for($i = 4; $i < sizeof($tables) - 2; $i = $i + 1){
```

Comment:
nit: Could be nice to have these constants either commented or abstracted, just in case things change in the future. it's not immediately clear to me what they're referencing